README.md

```diff
@@ -2,7 +2,19 @@
 
 # Fairness in machine learning - Mitigation algorithms
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning including:
+A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+
+## Current Release
+
+*  The current stable release is available at [FairLearn v0.2.0](https://github.com/microsoft/fairlearn/tree/v0.2.0 ).
+
```

Comment:
Remove extra space after * and extra space after v0.2.0 (the link doesn't work)

Reply:
done